horizon/enforcer/api.py

```diff
@@ -363,19 +363,19 @@ async def user_tenants(
         query: UserTenantsQuery,
         x_permit_sdk_language: Optional[str] = Depends(notify_seen_sdk),
     ):
-        response = await _is_allowed(query, request, USER_PERMISSIONS_POLICY_PACKAGE)
+        response = await _is_allowed(query, request, USER_TENANTS_POLICY_PACKAGE)
         log_query_result(query, response)
         try:
             raw_result = json.loads(response.body).get("result", {})
-            processed_query = (
-                get_v1_processed_query(raw_result)
-                or get_v2_processed_query(raw_result)
-                or {}
-            )
-
-            result = parse_obj_as(UserTenantsResult, raw_result.get("tenants", {}))
+            if isinstance(raw_result, dict):
+                allowed_tenants = raw_result.get("tenants", {})
+            elif isinstance(raw_result, list):
+                allowed_tenants = raw_result
```

Comment:
Will you be able to parse `allowed_tenants` if `raw_result` is list ?

Reply:
from now on it should be list